### ISSUE TITLE
When performing an adjust tour value after importing and saving a tour, the previous tour is reimported

### ISSUE THREAD

**Issue by FJBDev** [wolfgang-ch/mytourbook#249]
Tested on **20.3 & 20.8**.

Scenario:
1) **Import** a Tour (_Tour 1_) and **save** it in MT
2) Select a **different** tour (_Tour2_) already in the database and perform an action such as "`Compute Min/Max Temperature`", "`Compute Time Spent in each cadence zone`", `"Compute break time for all tours`" (there's probably more operation that trigger this issue but those ones will reproduce it for sure)
3) The _Tour1_ will be re-imported automatically but should NOT be re-imported as we are working with _Tour2_

:bug: ****Bug:** Why is the Tour1 re-imported when the user is performing an action on Tour2.** :bug: 

**EDIT: 2020-11-15**: After investigation, it seems that the Tour1 is re-imported because [this event was fired](https://github.com/wolfgang-ch/mytourbook/blob/95235426c2e81e6c15afb743eaa75fde326dd692/bundles/net.tourbook/src/net/tourbook/ui/views/rawData/DialogReimportTours.java#L652) and [triggered the re-import of Tour1](https://github.com/wolfgang-ch/mytourbook/blob/95235426c2e81e6c15afb743eaa75fde326dd692/bundles/net.tourbook/src/net/tourbook/ui/views/rawData/RawDataView.java#L1002)
Now the question becomes, should the re-import trigger this event `ALL_TOURS_ARE_MODIFIED` ?

Note: If the user restarts MT after step 1) and before step 2), the issue does not occur

```
18:30:31      Importing tours...
18:30:31   OK       11/3/20, 3:32 PM <- C:\Users\frederic\Desktop\2020-11-03_15-32-55.tcx
18:30:31   INFO       1 tours were imported from the file C:\Users\frederic\Desktop\2020-11-03_15-32-55.tcx
18:30:31      Imported in 0.119 s
18:30:34      Save tours
18:30:35   SAVE       11/3/20, 3:32 PM <- C:\Users\frederic\Desktop\2020-11-03_15-32-55.tcx
18:30:40      Loading selected tours...
18:30:40   INFO       1 tours loaded
18:30:40      Performed in 0.000 s
18:30:41      Computing min/max temperature values
18:30:41   OK       Updated tours: 1
18:30:41   ERROR       Tours without temperature values: 0
18:30:41      Performed in 0.007 s
```
!!!!!! SEE BELOW WHERE THE TOUR1 IS REIMPORTED!!!!!
```
18:30:41      Reimport previous files
18:30:42   OK       C:\Users\frederic\Desktop\2020-11-03_15-32-55.tcx
18:30:42      Reimported in 0.143 s

```

@wolfgang-ch 
I see this same issue in 20.11 when using the Re-Import feature:

- Import a Tour (Tour 1) and save it in MT
- Select a **different** tour (Tour2) already in the database and **re-import it** using the "Between dates" OR "All saved tours"
- The Tour1 will be re-imported automatically AND the Tour2 file will be imported too!

So it seems that any database operation after importing a file will reproduce this issue ?

**Comment by wolfgang-ch** [comment]
>  Now the question becomes, should the re-import trigger this event `ALL_TOURS_ARE_MODIFIED` ?

Yes this event must be triggered that views, e.g. tour chart can reload it's data but the question is if a reimport is necessary in [Import view](https://github.com/wolfgang-ch/mytourbook/blob/95235426c2e81e6c15afb743eaa75fde326dd692/bundles/net.tourbook/src/net/tourbook/ui/views/rawData/RawDataView.java#L1002) and I can't remember why a reimport is necessary

**Comment by FJBDev** [comment]
Ok but if the "between dates" is selected I don't think `ALL_TOURS_ARE_MODIFIED ` should be triggered but this one instead [TOUR_CHANGED](https://github.com/wolfgang-ch/mytourbook/blob/751942c6140c189a3b9924ba5f5d1241a5c67c31/bundles/net.tourbook/src/net/tourbook/tour/TourEventId.java#L29) ?

**Comment by FJBDev** [comment]
And I am curious to know why a re-import is necessary. If I compute the break time for all the tours, I don't see why re-importing the tours just imported is necessary but maybe at the time when you did it there was a valid reason

**Comment by wolfgang-ch** [comment]
I just found the commit where this code was introduced, 11 years ago.

For today, I close my computer, sometimes over night a solution appears, or not :-(

![mt-reimport](https://user-images.githubusercontent.com/1283445/99414897-8ea2d000-28f7-11eb-9b57-b8e01fb0d0d6.png)

For today I close my computer, sometimes over night a solution appears, or not :-(

**Comment by wolfgang-ch** [comment]
fixed in https://github.com/wolfgang-ch/mytourbook/commit/346f8a14f58ffb799a29e19768da00e723a7d448

**Comment by FJBDev** [comment]
> fixed in [346f8a1](https://github.com/wolfgang-ch/mytourbook/commit/346f8a14f58ffb799a29e19768da00e723a7d448)

That's a good solution, I might also use it after this is performed `"Compute Time Spent in each cadence zone"` as i don't think there is a need to reimport the files after computing the cadence zone times
